user/serializers.py

```diff
@@ -0,0 +1,32 @@
+from django.contrib.auth import get_user_model
+from rest_framework import serializers
+
+from .models import NotificationToken, Profile
```

Comment:
Try to use absolute imports instead of relative ones, they make things a little bit clearer to read. We actually use an addon to flake8 that enforces this, but I guess it wasn't set up on this repo